docs/release.adoc

```diff
@@ -3,9 +3,19 @@ include::.variables.adoc[]
 :toc: right
 :imagesdir: release/images
 
+
+== XP7 Update 14
+
+=== SDK
+
+* When a new XP sandbox is created using Enonic CLI 2.8 (or newer) and a template is chosen, the applications bundled with the chosen template are automatically installed during the sandbox's initial launch. This significantly streamlines the setup process, enabling developers to bypass the usual time-consuming application installation and dive straight into their work.
+* Welcome page (available at http://localhost:8080 in SDK version) displays the list of configuration files with an option to quickly copy their paths to the clipboard, enabling developers to quickly open the configuration files for modification in the Terminal.
+
+image:714-sdk-welcome.png["SDK Welcome page", 75%]
+
 == XP7 Update 13
 
-NOTE: Enonic XP 7.13.0 requires Content Studio 5.0.0
+NOTE: Enonic XP 7.13.0 requires Content Studio 5.0.0 or newer.
```

Comment:
XP does not require Content Studio - it makes no sense. So we are making this wrong sentence (introduced in 2895241c0242996f328c42f63680398af988378b ) even further